tooling/bundler/src/bundle.rs

```diff
@@ -161,7 +161,15 @@ pub fn bundle_project(settings: &Settings) -> crate::Result<Vec<Bundle>> {
         package_type: PackageType::Updater,
         bundle_paths: updater_paths,
       });
-    } else {
+    } else if updater.v1_compatible
+      || !package_types.iter().any(|package_type| {
```

Comment:
Hmm, maybe we need this to be dynamic depending on the currently built platform, but we could also add it later idc

Reply:
To be honest, the way this thing is currently structured is all about maintaining the old code path for v1 zipped updaters